src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderProfileDetail.java

```diff
@@ -53,10 +53,10 @@ public static class PK implements Serializable {
         private Integer sequence;
     }
 
-    @ManyToOne(optional = false)
+    @ManyToOne
     @JoinColumns(value = {
         @JoinColumn(name = "PROFILE_CODE", referencedColumnName = "PROFILE_CODE", insertable = false, updatable = false),
-        @JoinColumn(name = "PROFILE_TYPE", referencedColumnName = "PROFILE_TYPE", nullable = false, insertable = false, updatable = false)
```

Comment:
though type isn't nullable (and is mapped correctly in the primary key above), in this context it makes no sense to describe it as not  nullable when the entire link is nullable - and JPA won't allow it anyway